ckanext/publications_qld_theme/templates/header.html

```diff
@@ -91,11 +88,11 @@
       <nav class="section navigation">
         <ul class="nav nav-pills">
           {% block header_site_navigation_tabs %}
-          {% if c.controller == 'package' and c.id == 'publishing-standards-publications-qld-gov-au' %}
+          {% if h.current_url().startswith('/dataset/publishing-standards-publications-qld-gov-au') %}
```

Comment:
we may be able to rip this code out once 'cms' is in place on top of publications.